rrrspec-client/lib/rrrspec/client/rspec_runner_factory.rb

```diff
@@ -0,0 +1,24 @@
+require 'rrrspec/client/rspec_runner'
+
+module RRRSpec
+  module Client
+    class RSpecRunnerFactory
+      attr_reader :next_factory
```

Comment:
I think it's really hard to find a "general rule" based on a file extension. It's just a few characters. If there's no usage for that flexibility right now, it's better to keep it simple. Chaining is a good solution if the argument needs to be decorated (e.g. Java's Servlet API), but in this case it seems no use to me. If you should need more flexibility, it's better to let users to specify the factory directly and let them write in a composition style, instead of forcing chaining.